package.json

```diff
@@ -30,6 +30,7 @@
     "@skyux/packages": "^10.0.0",
     "@skyux/router": "^10.0.0",
     "@skyux/theme": "^10.0.0",
+    "@skyux/toast": "^10.44.2",
```

Comment:
Let's put this package at `"@skyux/toast": "^10.0.0",` to match the others.

Reply:
Done